sats-dna/src/main/kotlin/com/sats/dna/components/TrafficLights.kt

```diff
@@ -0,0 +1,82 @@
+package com.sats.dna.components
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.shape.CircleShape
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.unit.dp
+import com.sats.dna.theme.SatsTheme
+import com.sats.dna.tooling.LightDarkPreview
+
+@Composable
+fun TrafficLights(color: TrafficLightsColor, modifier: Modifier = Modifier) {
+    val actualColor = when (color) {
+        TrafficLightsColor.Green -> SatsTheme.colors.signal.success
```

Comment:
Maybe singular "TrafficLight" for both this composable and the color here as well? (Since it represents a single TrafficLight, not several ones)

Reply:
Oh, of course! :facepalm: